### PR TITLE
[15.0][MIG] website_sale_stock_available: Migration to 15.0 

### PR DESCRIPTION
**Pull request by chienandalu**
Adapt to the change in core that makes use of `free_qty` in eCommerce instead of `virtual_qty`. See:

- [IMP] website_sale_stock: use free_qty instead of virtual_quantity : https://github.com/odoo/odoo/commit/4da9533cdfea103e5a3082274b1b2a57cc67e16e
- [IMP] website_sale*: add to whislist and notify stock https://github.com/odoo/odoo/commit/88cb6db6de933908aaae4f15f2ebae167ad164ff#diff-9db95dd51674ba759a767208a1239bcbaeaa48a1b229867c0e4946a0313a6793

cc @Tecnativa TT37016

please review @ernestotejeda @sergio-teruel @pedrobaeza

### CODE REVIEW

File: website_sale_stock_available/README.rst [hunk 1]
```diff
@@ -14,36 +14,29 @@ Website Sale Stock Available
...
-This module extends the functionality of 'Product Availability' module
-(Technical name: website_sale_stock) so that for the 'Website shop' the
-'Available' quantity of a product is taken into account instead of
-'Forecasted' quantity.
+This module extends the functionality of the *Product Availability* module
+(technical name: ``website_sale_stock``) so that for the eCommerce the *Available*
+quantity of a product is taken into account instead of the *free* quantity.
 
-This image shows where you can see those quantities:
+Note that in the past the eCommerce availability was based in *Forcasted quantity*. This
```
Comment (pedrobaeza) [inline website_sale_stock_available/README.rst]: Little typo in forecasted

**Review by pedrobaeza — APPROVED** [review]
Cool icon.

File: website_sale_stock_available/README.rst (hunk 1 above)
Comment (chienandalu) [reply website_sale_stock_available/README.rst]: Solved

**Comment by chienandalu** [comment]
ping @sergio-teruel @ernestotejeda

**Review by ernestotejeda — APPROVED** [review]

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**